HUD/CreateMpGamerTagForPed.md

```diff
@@ -1,12 +1,12 @@
 ---
 ns: HUD
-aliases: ["0xBFEFE3321A3F5015"]
+aliases: ["0xBFEFE3321A3F5015", "_CREATE_MP_GAMER_TAG"]
 ---
-## _CREATE_MP_GAMER_TAG
+## _CREATE_MP_GAMER_TAG_FOR_PAD
 
 ```c
 // 0xBFEFE3321A3F5015 0xF5CD2AA4
-int _CREATE_MP_GAMER_TAG(Ped ped, char* username, BOOL pointedClanTag, BOOL isRockstarClan, char* clanTag, Any p5);
+int _CREATE_MP_GAMER_TAG_FOR_PAD(Ped ped, char* username, BOOL pointedClanTag, BOOL isRockstarClan, char* clanTag, int clanFlag);
```

Comment:
PAD ?

Reply:
Hups. Ped.

Reply:
The real name for this native was found actually.